spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogController.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.thymeleaf.blog;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.Random;
+
+@Controller
+public class BlogController {
+
+    @GetMapping("/blog/new")
+    public String newBlogPost(Model model)
+    {
```

Comment:
As per guidelines, the opening brace should be at the end of line 13, not by itself.

Reply:
Moved to same line